packages/cloudflare_logpush/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.36.0"
```

```suggestion
- version: "1.35.1"
```
Bugfixes bump patch.